src/main/app/form/validation.ts

```diff
@@ -260,6 +260,11 @@ export const isValidFileFormat = (files: any): boolean => {
   return AllowedFileExtentionList.indexOf(extension) > -1;
 };
 
+export const isExceedingMaxDocuments = (categoryKey: string, totalDocumentsLength: number): boolean => {
```

Comment:
Define the categorykey argument as optional one and assign "default" default value to it as an argument, so that you do not need to pass for default from the calling controllers.